main/FatesHistoryInterfaceMod.F90

```diff
@@ -3431,6 +3377,13 @@ subroutine update_history_dyn2(this,nc,nsites,sites,bc_in)
                    hio_area_si_landuse(io_si, cpatch%land_use_label) = &
                         hio_area_si_landuse(io_si, cpatch%land_use_label) &
                         + cpatch%area * AREA_INV
+
+                   hio_burnedarea_si_landuse(io_si, cpatch%land_use_label) = &
+                        hio_burnedarea_si_landuse(io_si, cpatch%land_use_label) + &
+                        cpatch%frac_burnt * cpatch%area * AREA_INV / sec_per_day
+
+                   hio_npp_si_landuse(io_si,cpatch%land_use_label) = hio_npp_si_landuse(io_si,cpatch%land_use_label) &
+                        + ccohort%npp_acc_hold * ccohort%n * dt_tstep_inv
```

Comment:
@ckoven since `FATES_NPP_LU` isn't in the high frequency section anymore, `dt_tstep_inv` should be removed for this, correct?  My understanding is that `_acc_hold` fluxes are in kgC/indiv/year.

```suggestion
                        + ccohort%npp_acc_hold * n_perm2 / (days_per_year*sec_per_day)
```

Reply:
Oh wait, this needs to be put further down in the history code since it's using a cohort level variable.  Somewhere below here?
https://github.com/NGEET/fates/blob/e118d40cc497fdfcc58021c714eb80ce569b11a2/main/FatesHistoryInterfaceMod.F90#L3482

Reply:
See 694e8064a4e717a51980d62fd49746c0f858c938 for suggested fix.

Reply:
Yes, 694e806 looks correct to me.